src/controllers/User.controller.js

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable max-len */
+/* eslint-disable no-console */
+'use strict';
+
+const UserService = require('../services/User.Service');
+
+const get = async(request, response) => {
+  try {
+    const allUsers = await UserService.getAll();
+
+    if (!allUsers) {
+      response.status(404).send('Not Found: The specified entity does not exist');
+    }
+
+    response.statusCode = 200;
+    response.send(allUsers);
+  } catch (error) {
+    console.log(error);
+    response.status(500).send('Internal Server Error');
+  }
+};
+
+const getOne = async(request, response) => {
+  try {
+    const { id } = request.params;
+    const user = await UserService.getOne(id);
+
+    if (user === null) {
+      response.sendStatus(404);
+
+      return;
+    };
+
+    response.statusCode = 200;
+    response.send(user);
+  } catch (error) {
+    console.log(error);
+    response.status(500).send('Internal Server Error');
+  }
+};
+
+const create = async(request, response) => {
+  try {
+    const { name } = request.body;
+    const createdUser = await UserService.create(name);
+
+    if (createdUser === null) {
+      response.sendStatus(400);
+
+      return;
+    }
+
+    response.statusCode = 201;
+    response.send(createdUser);
+  } catch (error) {
+    console.log(error);
+    response.status(500).send('Internal Server Error');
+  }
+};
+
+const update = async(request, response) => {
+  try {
+    const { id } = request.params;
+    const updatedUser = await UserService.update(request.body.name, id);
```

Comment:
Avoid using 'request.body.name' in the 'update' function. Instead, pass the whole 'request.body' as a parameter to the 'UserService.update' method. This will make the function more generic and allow you to update other properties of the user in the future without changing the controller code.